### PR TITLE
Add patient

### PR DESCRIPTION
**Pull request by mohamed-ahmed-yousef**

### CODE REVIEW

File: app/[locale]/actions/patient.ts
Comment (assawalhy) [inline app/[locale]/actions/patient.ts:1]: I suggest removing `"use server"` and using these async function directly in the client side.

File: app/[locale]/types/patient.ts
Comment (assawalhy) [inline app/[locale]/types/patient.ts:1]: Move to `@/lib/types.ts`

Comment (mohamed-ahmed-yousef) [reply app/[locale]/types/patient.ts:1]: Ok

File: app/[locale]/actions/patient.ts
Comment (mohamed-ahmed-yousef) [reply app/[locale]/actions/patient.ts:1]: ok, I suggest creating a folder for API calls, do u think it's a better way?

**Comment by mohamed-ahmed-yousef** [comment]
Please merge if all thing LGTY @MuhammadSawalhy

**Review by assawalhy — APPROVED** [review]
Great

**Pull request merged**